File: Java/LeapYear.java
import java.util.Scanner;

public class LeapYear{

    public static void main(String[] args) {
        
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter a Year : ");
        int year = scan.nextInt();

        if(year%4==0 || year%100==0)
            {
                System.out.println("Yes it is a Leap year");
            }
        else
            {
                System.out.println("No it is not a Leap year");
            }

    }

}

File: Python/LeapYear.py
year = int(input("Enter Year: "))
if year%4!=0:
    print(year, "isn't a Leap Year")
elif year%100==0 and year%400!=0:
    print(year, "isn't a Leap Year")
else:
    print(year, "is a Leap Year")
